corehq/apps/style/static/preview_app/less/preview_app/form.less

```diff
@@ -40,12 +40,7 @@
 }
 
 .form-single-question {
-  padding-top: 66px;
-  min-height: 443px;
-}
-
-.preview-tablet-mode .form-single-question {
-  min-height: 642px;
```

Comment:
ah ok, i thought that's what it was doing, but if it's too large it makes the screen for OQPS scrollable